Sources/SQLiteNIO/SQLiteDataConvertible.swift

```diff
@@ -123,6 +123,12 @@ extension Date: SQLiteDataConvertible {
             value = v
         case .integer(let v):
             value = Double(v)
+        case .text(let v):
+            guard let d = dateTimeFormatter.date(from: v) ?? dateFormatter.date(from: v) else {
```

Comment:
I'm not a huge fan of the "try one and then the other" approach this requires, but I can't think of a good alternative either.